### PR TITLE
Add Fritz! logo (SVG)

### PR DESCRIPTION
**Pull request by logiclrd**
This PR:
- Adds a logo for Fritz! (FRITZ!Box), which Wikipedia says is a manufacturer of consumer Internet modems (DSL, Cable, 3G/LTE, fibre) in Germany.
- Adds the logo to README.md.
- Adds a reference image and updates the reference index.md correspondingly.

### CODE REVIEW

**Comment by logiclrd** [comment]
Resolves #363.

File: images/svg/fritz.svg [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"
+aria-label="Fritz!" role="img"
+viewBox="0 0 512 512"><rect
+width="512" height="512"
+rx="15%"
+fill="#F8EC17"/><style>.c{fill:#e2001a}.b{fill:#000}</style><use xlink:href="#a" x="4" y="3.5" class="b" /><g class="c"><path id="a" d="m246 298h-34v-88l13-3-4-16h25zm-127 26h33v-87h6c3 0 10 4 10 12 0 10-9 17-16 23l34,56c9-7 19-16 26-24l-27-32c14-11 17-20 16-30-2-21-21-25-36-27-12-1-33-1-45-0zm194 0v-83l28 1h2v-30h-93v30h2l28 -1v83zm66-109l-43,82c32 0 54 0 88 0v-30c-18 1-30 1-40 0l43-80c-38-2-65-2-84-2v30c15-1 26-1 36-1zm-269-28c-27 0-50 1-66 1v119h34v-42h29v-28h-29v-21h32v-10zm350 74l8.5-56-28-2-3.5 57zm-15 33l18-15-15-18-18 15z" stroke="#000" stroke-width="2.6" /></g></svg>
```
Comment (edent) [inline images/svg/fritz.svg]: You can save a few bytes here. 
` stroke-width="2.6" /></g>`  can become `stroke-width="2.6"/></g>`  (remove the surplus space.)
I think there are a couple of those which can be squashed.

**Comment by edent** [comment]
Really great work! Can you remove a couple of extra spaces please?

**Comment by logiclrd** [comment]
These are all going to conflict in the same way, aren't they? :-P Is it okay to leave sorting that out to you?

**Comment by edent** [comment]
Yeah, I'll futz the conflicts somehow :-)

**Comment by logiclrd** [comment]
For the Samsung one, I just rebased it onto the new `master`, and the GUI diff tool I have has the option to right-click a line and say "use the right side's version followed by the left side's version (keep both)", after which I did a force push. That minimizes the chance of me fat-fingering something manually editing a file with merge conflict markers in it, and keeps the commit history nice and clean :-)

**Comment by logiclrd** [comment]
Sweet, thanks :-D

**Pull request merged**